src/model/cloud-runner/services/cloud-runner-custom-steps.ts

```diff
@@ -43,25 +43,91 @@ export class CloudRunnerCustomSteps {
     aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID --profile default
```

Comment:
I've seen quite a few `aws` cli usage passed to github actions steps directly like that. I was wondering if it would make more sense to start using https://www.npmjs.com/package/aws-sdk and some more high level code for some of these steps or it would be too complicated.

I assume this shouldn't be done as part of this pr, but could be interesting to explore as code is easier to test than shell scripts hardcoded in yaml :)

Reply:
Currently we save ourself the bundle size of these sdks. Easier for me to include if we prefer. 🤷